tools/scripts/generic.mk

```diff
@@ -218,6 +218,9 @@ ifneq ($(REL_ASM_SRCS),$(ASM_OBJS_S))
 	ASM_OBJS += $(ASM_OBJS_S)
 endif
 
+SRCS_IN_BUILD = $(call relative_to_project, $(SRCS))
```

Comment:
I wanted to do this but this way:
![image](https://user-images.githubusercontent.com/16192750/108037544-6cc60000-7042-11eb-88c1-0721a89853ff.png)

else you have to move olaso INCS:= $(sort $(INCS)) and SRCS = $(sort $(SRCS))